tests/QueryBuilderTest.php

```diff
@@ -617,6 +619,43 @@ public function testSubdocumentArrayAggregate()
         $this->assertEquals(12, DB::table('items')->avg('amount.*.hidden'));
     }
 
+    public function testAggregateGroupBy()
+    {
+        DB::table('users')->insert([
+            ['name' => 'John Doe', 'role' => 'admin', 'score' => 1],
+            ['name' => 'Jane Doe', 'role' => 'admin', 'score' => 2],
+            ['name' => 'Robert Roe', 'role' => 'user', 'score' => 4],
+        ]);
+
+        $results = DB::table('users')->groupBy('role')->orderBy('role')->aggregateByGroup('count');
+        $this->assertInstanceOf(LaravelCollection::class, $results);
+        $this->assertEquals([(object) ['role' => 'admin', 'aggregate' => 2], (object) ['role' => 'user', 'aggregate' => 1]], $results->toArray());
+
+        if (! method_exists(Builder::class, 'countByGroup')) {
+            $this->markTestSkipped('countBy* function require Laravel v11.38+');
```

Comment:
Based on the description of [PHPORM-286](https://jira.mongodb.org/browse/PHPORM-286), I assume it's the "*ByGroup" functions that require Laravel 11.38+.

Also fixes subject/verb agreement.

```suggestion
            $this->markTestSkipped('*byGroup functions require Laravel v11.38+');
```